.github/policies/pr.triage.yml

```diff
@@ -0,0 +1,125 @@
+# This file is generated by the sync-labels script DO NOT EDIT manually
+id: pr.triage
+name: Assign area labels to PRs
+description: Assign area labels to PR depending on path modified.
+resource: repository
+disabled: false
+configuration:
+  resourceManagementConfiguration:
+    eventResponderTasks:
+      - description: Assign area labels to PRs based on modified files
+        if:
+          - payloadType: Pull_Request
+          - isOpen
+        then:
+          - if:
```

Comment:
We have this mapping in code as well of area to label, any chance we can change this yml to call some code that uses the same definition so we don't have it in two places?

Reply:
same as above